src/server/visual-diff-plugin.js

```diff
@@ -1,6 +1,8 @@
-import { access, constants, mkdir, rename, stat } from 'node:fs/promises';
-import { dirname, join } from 'node:path';
+import { access, constants, mkdir, readdir, rename, rm, stat } from 'node:fs/promises';
+import { basename, dirname, join } from 'node:path';
+import { env } from 'node:process';
 
+const isCI = !!env['CI'];
```

Comment:
Added this to make CI as quick as possible, but we could leave it out if we'd rather keep things simple.  It'll just search for `fail`, `pass`, and `reports` that won't exist.